### DISCUSSION TITLE
Updating version of specific camel component

### DISCUSSION THREAD

**Discussion by dpalinic** [apache/camel-karavan#1015] [General]
Hello,

I'm wondering if it's possible to update only one camel component to the higher version when working with a development installation on local laptop? There is a bug in camel-http which is solved in camel-http version 4.2.0 (and 4.0.3) and we would like to load fixed version, without a need to wait for Karavan 4.2.0 version.

We've tried to load it in application.properties inside of the camel.jbang.dependencies property, but it's not working for camel libraries (I guess to preserve the same version across the system).

Thanks

**Comment by davsclaus** [comment]
You can set the camel version to run and use for build and deploy. But the UI is fixed on a given camel release.
In VSCode there is a setting to set the camel version.
